test/testdata/desugar/forward_args.rb.desugar-tree.exp

```diff
@@ -0,0 +1,29 @@
+class <emptyTree><<C <root>>> < (::<todo sym>)
+  class <emptyTree>::<C Foo><<C <todo sym>>> < (::<todo sym>)
+    def bar<<C <todo sym>>>(a, b, k1:, k2:, &block)
+      <emptyTree>
+    end
+
+    def foo<<C <todo sym>>>(*<args>, *<kargs>:, &<blk>)
```

Comment:
It's really a `parser::Kwrestarg` but it's displayed like `*name`

Reply:
it's displayed like `*<kargs>:` haha the trailing `:` means that it's wrapped correctly.

This is a confusing printing; we could definitely special case this in the `toString` method.

Reply:
In the middle of the `desugar-tree` output I *never* noticed the `:`, thanks!